docs/lesson/improvements.rst

```diff
@@ -0,0 +1,290 @@
+.. _improvements:
+
+Improving a lesson
+====================
+
+In this section, you will learn how to make your exploration better by:
+
+ * :ref:`Using the Improvements tab <improvementtab>`
+         - :ref:`Getting feedback <feedback>`
+         - :ref:`Suggestions for improvement <suggestions>`
+         - :ref:`Answer details for a card <answerdetails>`
+         - :ref:`Exiting early <earlyexit>`
+         - :ref:`Multiple incorrect answers <incorrectanswer>`
+         - :ref:`Getting stuck in a loop <cyclicloop>`
+ * :ref:`Adding branches to an exploration <addbranch>`
+
+Watch this video to learn how to improve your exploration, or if you prefer reading the instructions instead, keep scrolling down.
+
+.. raw:: html
+
+   <iframe width="560" height="315" src="https://www.youtube.com/embed/fjQdl6i0vu0" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+.. _improvementtab:
+
+Using the Improvements tab
+***************************
+
+Oppia's explorations are designed to be improved over time as learners engage with the content. You can use the Improvements tab to determine how learners are using your exploration and what they think about it, and make changes accordingly.
+
+1. Click the |improvements| **Improvements** button in the navigation menu. 
+2. At the very top of the page, you can see how many **Open Tasks** are waiting for your review. 
+
+The following are the types of open tasks you might see.
+
+.. |improvements| image:: /images/improvements_tab.png
+                  :alt: Improvements button
+                  :scale: 30 %
+
+.. _feedback:
+
+Feedback on a card
+-------------------
+
+Learners have the option to provide feedback on each card as they playthrough an exploration, as well as at the end when they are asked to review the exploration. If they have provided any feedback, you will see a task similar to the image below.
+
+.. figure:: /images/feedback_task.png
+   :alt: Feedback task
+   :scale: 40 %
+
+   *Fig. 1*
+
+1. To change the status of the feedback or write a message in response to the feedback for your record, click on the |reviewthread| button.
+2. Click |sendandclose| when done.
+
+.. caution:: 
+   Leave the status at *Open* if you are still working on the task. Only use the other options when you are ready to remove the task from the Improvements screen.
```

Comment:
Let's say "hidden" instead, remove implies (to me) that they are gone forever. Resolved tasks can be viewed again by unticking the "only show open tasks" checkbox at the top of the improvements tab page.

Reply:
Done